admin-dev/themes/new-theme/js/pages/cms-page/index.js

```diff
@@ -65,4 +66,11 @@ $(() => {
   shopChoiceTree.enableAutoCheckChildren();
 
   new TranslatableInput();
+
+  new TaggableField({
```

Comment:
This is so cool that we can reuse our components so easily 😊 it means the job was well done

👏

Reply:
yeah indeed , took me whole minute to add taggable field :D